_pages/profile/submit/submit-profile/finalize-tab.js

```diff
@@ -75,6 +75,7 @@ class _FinalizeTab extends React.Component {
           <Col span={24}>
```

Comment:
`About me` on the finalize-tab is black and feels weird. Text should be white

Reply:
Now I realize what you and @bilinkis meant. Here's a quick fix: https://github.com/OpenProofOfHumanity/proof-of-humanity-web/pull/6/commits/8579239a781d3b9df5ec8756830c3b22de1d0d66 
<img width="941" alt="Screen Shot 2022-11-21 at 5 14 14 PM" src="https://user-images.githubusercontent.com/2153166/203104466-55da4445-6650-4623-b0a0-3b13539c9814.png">